libs/sdk-common/src/input_parser.rs

```diff
@@ -628,6 +648,8 @@ pub enum InputType {
     #[cfg(feature = "liquid")]
     Bolt12Offer {
         offer: LNOffer,
+        /// The BIP353 address in case one was resolved
```

Comment:
This, to me, reads like BIP353 address is something resolved from payment metadata. Perhaps it's better to denote the payment was originated from a BIP353 Address.

For example:
```suggestion
        /// The BIP353 address from which this payment was resolved.
```

Reply:
I improved it in [e6cea9b](https://github.com/breez/breez-sdk-greenlight/pull/1164/commits/e6cea9b08c75364ea10fbc6e3cd4f925d1aabaa4). Thanks